doc_swift_ui_gallery/swift_ui_gallery/swift_ui_gallery/layouts/ZStackExamples.swift

```diff
@@ -0,0 +1,155 @@
+import SwiftUI
+
+struct ZStackDemo: View {
+    var colors: [Color] = [.red, .orange, .yellow, .green, .blue, .indigo, .purple]
+    var body: some View {
+        NavigationStack {
+            ZStack {
+                ForEach(0..<colors.count) {
+                    index in
+                    Rectangle()
+                        .fill(colors[index])
+                        .frame(width: 100, height: 100)
+                        .offset(x: CGFloat(index) * 10.0, y: CGFloat(index) * 10.0)
+                }
+            }
+        }.navigationTitle("ZStack Demo")
+    }
+}
+
+struct ZStackTopAlignment: View {
+    var body: some View {
+        NavigationStack {
+            ZStack(alignment: .top) {
+                Rectangle().fill(.red).frame(width: 200, height: 200)
+                Rectangle().fill(.green).frame(width: /*@START_MENU_TOKEN@*/100/*@END_MENU_TOKEN@*/, height: 100)
```

Comment:
What do "@START_MENU_TOKEN@" and "@END_MENU_TOKEN@" mean?

Reply:
It was added by Xcode. Have removed it.